### DISCUSSION TITLE
Copilot.vim

### DISCUSSION THREAD

**Discussion by ecthelionvi** [community/community#44058] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have been using Copilot with neovim for a while now flawlessly but today I started getting this popup every time I go to open up a file. Copilot is authenticated and setup, works properly but I do not know why I keep seeing this message:

`Request Actions:
1. Signup for GitHub Copilot
2. Dismiss
(Type number and <Enter> or click with the mouse (q or empty cancels):`

**Comment by iamcjwilliams** [comment]
Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot. If you continue to have issues, please provide screenshots
